evm_arithmetization/src/cpu/kernel/constants/global_metadata.rs

```diff
@@ -112,7 +115,7 @@ pub(crate) enum GlobalMetadata {
 }
 
 impl GlobalMetadata {
-    pub(crate) const COUNT: usize = 55;
+    pub(crate) const COUNT: usize = 56;
```

Comment:
This can be private I think.

Reply:
As a general comment regarding these types of suggestions, while I agree, I think it may be cleaner if we were to address all of them in a `cleanup`-like PR, so as to not interfere with the actual targeted scope. Otherwise it may get a bit messy for reviewers.